### PR TITLE
Added support for Postgresql

### PR DESCRIPTION
**Pull request by joshpfosi**

### CODE REVIEW

**Comment by jakesutton** [comment]
:+1:

**Comment by joshpfosi** [comment]
So I've gone and really mucked up these PRs lol. Please take them both or none at all. Just noticed that this one contains changes from the other one which will cause specs to fail... Sorry!

**Comment by artfuldodger** [comment]
Awesome. Thank you.

File: rspec_profiling.gemspec [hunk 1]
```diff
@@ -20,6 +20,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency "sqlite3"
   spec.add_dependency "activerecord"
+  spec.add_dependency "pg"
```
Comment (artfuldodger) [inline rspec_profiling.gemspec:23]: Hm. Sort of a bummer to require both `sqlite3` and `pg` now..

**Comment by artfuldodger** [comment]
Two things..
1. Would you mind rebasing so I can just click the button? :smiley: 
2. What do you think about moving the impetus for requiring `sqlite3`/`pg` to the user and just documenting that in the README? I think I prefer that to having both as dependencies, but I could probably be persuaded otherwise.

And a bonus third thing: again, thank you for your contributions to this gem. Really awesome.

**Comment by joshpfosi** [comment]
There ya go :) Ready for merging! I would prefer moving the dependency but not sure if that would work correctly. I suppose it would.... I do like that better.... Hmm.

**Comment by artfuldodger** [comment]
Thanks! I'll create an issue for addressing the superfluous gem dependencies. I'll go ahead and bump the version and publish.

**Pull request merged**